.github/workflows/test.yaml

```diff
@@ -24,4 +24,16 @@ jobs:
           haproxy -vv
 
       - name: Test
-        run: go run mage.go test
\ No newline at end of file
+        run: go run mage.go test
```

Comment:
I would run this in a different workflow instead of the normal test one

Reply:
Done